app/categories/domestic_abuse/forms.py

```diff
@@ -1,27 +1,5 @@
-from app.categories.forms import QuestionForm
-from wtforms import RadioField
-from app.categories.widgets import CategoryRadioInput
-from wtforms.validators import InputRequired
+from app.categories.forms import SafeguardingQuestionForm
 
 
-class AreYouAtRiskOfHarmForm(QuestionForm):
+class AreYouAtRiskOfHarmForm(SafeguardingQuestionForm):
```

Comment:
Can we rename this class, as the question no longer asks if the user is at risk of harm.

Can I suggest `WorriedAboutSomeonesSafetyForm`? Just so it's clear what this question is asking.

Reply:
Renamed to `WorriedAboutSomeonesSafetyForm`